.gitignore

```diff
@@ -1 +1,2 @@
 argocd-strongbox-plugin
+testData/app-with-secrets-test
```

Comment:
Worth adding all of `testData/` to ignore?

Reply:
there are 2 other folder in testData we need those to run tests.
basically its copies `app-with-secrets` as starting point and run test and delete test dir.